classes/mock/generator.php

```diff
@@ -721,6 +721,7 @@ protected function getParametersSignature(\reflectionMethod $method, $forceMockC
                     $parameterCode .= ' = ' . var_export($parameter->getDefaultValue(), true);
                     break;
 
+                case $parameter->allowsNull():
```

Comment:
well thanks for the code, I update the PR (and add a light improvement, we need to check that the default value != null ;)

by the way, don't you think it could be a nice issue to submit to php to have a isNullable() ?